### ISSUE TITLE
Show informative message when wordwrap doesn't work for large file

### ISSUE THREAD

**Issue by burtonsys** [Microsoft/vscode#45596]
Issue Type: <b>Bug</b>

Open a large text file (296,847 lines, 21 MB, UTF8, no BOM, suffix ".txt"). Even though "editor.wordWrap": "on" is configured, long lines do not wrap.

Pressing Alt-Z does not cause long lines to wrap, though it does slightly change the width of the left part of the horizontal scrollbar.

It worked fine until yesterday. When I first noticed the problem Visual Studio Code was prompting me to install the latest update, but after the update was installed the problem still occurred.

This text file has been slowly growing over the years, so it is possible that the problem was triggered by the growth of the file.

VS Code version: Code 1.21.0 (9a199d77c82fcb82f39c68bb33c614af01c111ba, 2018-03-07T11:04:09.969Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-2310 CPU @ 2.90GHz (4 x 2893)|
|Memory (System)|15.91GB (2.26GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none
Reproduces without extensions

**Comment by burtonsys** [comment]
How's it going, alexandrudima? Any progress? Have you managed to reproduce it? Do you need help with that?

**Comment by alexandrudima** [comment]
We have a hard-coded size of 20MB, above which files will get certain features disabled (to save up on memory usage). There is a PR to make this value configurable

**Comment by alexandrudima** [comment]
@burtonsys The insiders build has merged in Pull Request #44988 which makes the size configurable.

You can download the insiders build from https://code.visualstudio.com/insiders/ and configure `editor. largeFileSize` or `editor. largeFileLineCount` with higher values.

**Comment by burtonsys** [comment]
**_Thank you!!!_**

I've configured the new version like this:

{ "files.autoGuessEncoding": true,
    "editor.wordWrap": "on",
    "window.zoomLevel": 1,
    "editor.accessibilitySupport": "off",
    "git.ignoreMissingGitWarning": true,
    "editor.largeFileSize":41943040,
    "editor.largeFileLineCount": 600000,
}

At first it appeared that that hadn't solved the problem. Alt-Z just changed the width of the "button area" of the horizontal scrollbar, and didn't actually make wordwrap work. But I closed the big file, and then reopened it, and then it worked.

I still would like to suggest that VSCode should display a warning message, with instructions for what to do about the problem, when those limits are exceeded.
